cmd/gbounty/bootstrap/bootstrap.go

```diff
@@ -592,13 +610,13 @@ func finalizeScan(ctx context.Context, updatesChan chan *scan.Stats, cfg scan.Co
 			storeOutput(ctx, cfg, fs)
 
 			// If no silent, we print the summary as well.
-			if !cfg.Silent {
+			if !cfg.Silent || !PocEnabled {
```

Comment:
If that's the desired behavior (I'm not sure), I'd try to "join" `cfg.Silent` with `PocEnabled`, so we print what we want, but re-use the same conditional statement for the rest of the places.